packages/suite-web/src/support/BrowserDetect/index.tsx

```diff
@@ -0,0 +1,46 @@
+// No styled components here, this is supposed to be a very low dependency component
+import React, { useMemo } from 'react';
+import UAParser from 'ua-parser-js';
+import { supportedBrowsers } from './constants';
+
+import Unsupported from './components/Unsupported';
+import NoWebUSB from './components/NoWebUSB';
+import Outdated from './components/Outdated';
+import ChromeAndroid from './components/ChromeAndroid';
+
+interface Props {
+    children: any;
+}
+
+const BrowserDetect = ({ children }: Props) => {
+    const ua = useMemo(() => new UAParser().getResult(), []);
```

Comment:
I'm no expert  but isn't
> Don't use user agent, use feature detection

a [widely spread principle](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)?

Reply:
Not maintainable here, as you would have to check hundreds of features

Reply:
I feared so. .)